compiler/damlc/daml-stdlib-src/DA/Internal/LF.daml

```diff
@@ -243,6 +243,26 @@ data AnyContractKey = AnyContractKey
   , getanyContractKeyTemplateRep : TemplateTypeRep
   }
 
+#ifdef DAML_GENMAP
```

Comment:
Why is this guarded by the existence of `DAML_GENMAP` and not by something related to the existence of the `Any*` types?

Reply:
Because we need the generic equality operation for this to work and we don’t separate that from generic maps. Technically it needs to be guarded by both but that’s redundant since we never have a case where we support generic equality but do not support `Any`. I’ve added it anyway to make it more clear and I also added a comment.